docs/kb/developer-resources/running-commands-on-the-host-os.md

```diff
@@ -25,7 +25,7 @@ name: "Application"
 properties:
   logo_url: http://www.replicated.com/images/logo.png
   console_title: Application
- 
+
```

Comment:
The changes to the block text work ok in the yaml editor as well as in the generated file inside the container.